docs/resources/widgets/Concepts/composing-widgets.md

```diff
@@ -0,0 +1,136 @@
+---
+title: Composing Widgets
+sidebar_position: 2
```

Comment:
I feel like this page looks a bit weird. The images take up a lot of space and theres a lot of blank space. I think it makes sense to put this stuff into a tabular format and make the images much smaller so it's easier to view

Reply:
Like I would have a table with row, stack, column each as their own row and the little images you put as a column in the table